ZEP/instructions/zep0000.md

```diff
@@ -0,0 +1,227 @@
+require collaboration across multiple projects.
+
+## ZEP types
+
+> Specification ZEP
+
+Specification ZEPs deal with the changes related to Zarr spec. The SPEC ZEP could introduce changes in core specification or can add changes as an extension to the core protocol. The changes related to core specification follow a strict and thorough review process and should be adopted by everyone in the Zarr community, whereas changes made as an extension to the core protocol follow a comparatively simple review process and is not strictly necessary to be adopted by everyone in the Zarr community. The specification ZEPs are of two types:
+
+- Core protocol ZEP
+
+Describes a ZEP which involves changes in the core specification protocol for Zarr. Core protocol ZEPs(commonly known as Core ZEPs) are a part of Spec ZEPs and apply to [Zarr storage specification](https://github.com/zarr-developers/zarr-specs) and its various implementations under the [Zarr Implementations](https://github.com/zarr-developers/zarr_implementations) GitHub repo. The core protocol ZEPs should be adopted by every implementation of Zarr and, in general, the overall Zarr community. Core ZEPs must go through a thorough review process, including involvement, discussion, and voting from the author of the proposed ZEP, Zarr Steering Council, author of various Zarr implementations, and open-source projects/research groups using Zarr and the general Zarr community. The general advice is that everyone should be made aware of the change by the introduction of core ZEP. Core protocol ZEP require community consensus, and users are typically not free to ignore them.
+
+- Protocol extension ZEP
+
+Describes a ZEP which involves changes in the extension of the core specification protocol for Zarr. Protocol extension ZEPs(commonly known as extension ZEPs) are part of Spec ZEPs and apply to Zarr storage extension specification and various implementations under the Zarr Implementations GitHub repo. The extension ZEPs are designed so that they are not necessary to be adopted by every other Zarr implementation or by the whole community. They follow a lenient review process compared to core ZEPs, which includes involvement, discussions, and voting from the author of the proposed ZEP, Zarr Steering Council, and open-source/research groups affected by the change. The extension ZEPs may or may not include involvement from the author of various Zarr implementations and the general Zarr community. The general idea of having an extension ZEP is to add functionality, data types and features in a much more relaxed and quick manner to accommodate emerging and essential use cases. The general advice is that the Zarr community and anyone else who will be affected by the extension ZEPs should be made aware of the change. Extension ZEPs may or may not require community consensus, and it is up to users to ignore them.
```

Comment:
We should clarify what we mean by users or use a different term. Maybe implementers if that is what is intended?